rct229/rules/section5/section5rule47.py

```diff
@@ -0,0 +1,87 @@
+from rct229.rule_engine.rule_base import RuleDefinitionListIndexedBase, RuleDefinitionBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.ruleset_functions.get_surface_conditioning_category_dict import (get_surface_conditioning_category_dict, SurfaceConditioningCategory as SCC)
+from rct229.ruleset_functions.get_zone_conditioning_category_dict import (get_zone_conditioning_category_dict, ZoneConditioningCategory as ZCC)
+from rct229.schema.config import ureg
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import ZERO
+from rct229.utils.std_comparisons import std_equal
+
+TARGET_AIR_LEAKAGE_COEFF = 1.0 * ureg("cfm / foot**2")
+TOTAL_AIR_LEAKAGE_COEFF = 0.112
+
+
+class Section5Rule47(RuleDefinitionListIndexedBase):
+    """Rule 47 of ASHRAE 90.1-2019 Appendix G Section 5 (Envelope)"""
+
+    def __init__(self):
+        super(Section5Rule47, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            required_fields={
+                "$": ["weather"],
+                "weather": ["climate_zone"],
+            },
+            each_rule=Section5Rule47.BuildingRule(),
+            index_rmr="baseline",
+            id="5-47",
+            description="The baseline air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 1 cfm/ft2. The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4.",
+            list_path="ruleset_model_instances[0].buildings[*]",
+        )
+
+    def create_data(self, context, data=None):
+        rmr_proposed = context.proposed
+        return {"climate_zone": rmr_proposed["weather"]["climate_zone"]}
+
+    class BuildingRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section5Rule47.BuildingRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, False, True),
```

Comment:
This rule only uses the baseline, so the arguments should be `(False, True, False)`.